core/server/master/src/main/java/alluxio/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -132,16 +132,16 @@ public void checkConsistency(CheckConsistencyPRequest request,
       StreamObserver<CheckConsistencyPResponse> responseObserver) {
     String path = request.getPath();
     CheckConsistencyPOptions options = request.getOptions();
-    RpcUtils.call(LOG,
-        (RpcUtils.RpcCallableThrowsIOException<CheckConsistencyPResponse>) () -> {
-          List<AlluxioURI> inconsistentUris = mFileSystemMaster.checkConsistency(
-              new AlluxioURI(path), CheckConsistencyContext.create(options.toBuilder()));
-          List<String> uris = new ArrayList<>(inconsistentUris.size());
-          for (AlluxioURI uri : inconsistentUris) {
-            uris.add(uri.getPath());
-          }
-          return CheckConsistencyPResponse.newBuilder().addAllInconsistentPaths(uris).build();
-        }, "CheckConsistency", "path=%s, options=%s", responseObserver, path, options);
+    RpcUtils.call(LOG, (RpcUtils.RpcCallableThrowsIOException<CheckConsistencyPResponse>) () -> {
+      List<AlluxioURI> inconsistentUris =
+          mFileSystemMaster.checkConsistency(mFileSystemMaster.translateUri(path),
```

Comment:
Instead of doing this inline, could we do it as part of declaring `path`?

Reply:
Done. I took the chance to do some refactoring that I wanted to do. 
In case it's confusing, I done below additionally:
- Simplify lambda syntax
- Unify RPC description by printing request only.

Reply:
Please note that, I don't translate every URI coming to this handler. Specifically for `mount` / `umount` / `start-sync` / `stop-sync` / `schedule-persist`.

If you see any problem with that within transparent URI scenario, I could still do the translation. However I don't expect to see non-Alluxio URIs coming in for these RPCs as they are not used by HDFS client.

Reply:
Discussed with @madanadit offline. I'll unify all these, here and at `RetryHandlingFileSystemMasterClient`

Reply:
Actually `mount/unmount` can't be modified as translating a mount path will simply fail. Now with these mandatory exclusions I feel it's OK to not manipulate paths for RPCs that are not accessed via hadoop. @madanadit ?

Reply:
we can still unify path manipulation logic in`RetryHandlingFileSystemMasterClient` though.